Cargo.toml

```diff
@@ -64,9 +82,16 @@ async-std = { version = "1.9.0", default-features = false, optional = true }
 # Feature: lobby
 md-5 = { version = "0.10", optional = true }
 uuid = { version = "0.7", optional = true, features = ["v3"] }
+nix = "0.26"
```

Comment:
Move the `nix` import above the feature sections to make clear it's not part of a specific feature.

Reply:
Oh I used `cargo add` and I guess it just threw it in wherever